src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,62 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { SetStateAction } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  setFilterTodoBy: React.Dispatch<React.SetStateAction<string>>,
+  filterTodoBy: string,
```

Comment:
it would be better to add `enum FilterBy` and use it everywhere
```suggestion
  filterTodoBy: FilterBy,
```